broadinstitute/ddp-study-server
# DDP-8628: supporting human-readable export format

We would like to give the users an explicit choice between human readable and analysis friendly formatting.  This adds a single new variable "humanReadable" which consolidates a choice between splitting multiselect questions into multiple columns, and rendering option stable ids vs. the display text.  This also merges some bugfixes and test improvements from develop

This is paired with  https://github.com/broadinstitute/ddp-angular/pull/1658

TO TEST:
1. open the participant list for the AT study
2. customize view to include the Medical History "telangiectasia" question
3. export the participant list, selecting "analysis-friendly" 
4. confirm the telangiectasia responses are split into 4 columns - 1 each for eyes and skin, and 1 for each of the supporting details
5. export the participant list again, selecting "human-readable"
6. confirm the telangiectasia responses are shown in two columns -- one showing the main responses as dispaly text, and the other showing the consolidated details
